SPECS/edk2/edk2.spec

```diff
@@ -337,7 +347,9 @@ cp -a -- \
    %{SOURCE30} %{SOURCE31} %{SOURCE32} \
    %{SOURCE40} %{SOURCE41} %{SOURCE42} %{SOURCE43} %{SOURCE44} \
    %{SOURCE45} %{SOURCE46} %{SOURCE47} %{SOURCE48} \
-   %{SOURCE80} %{SOURCE81} %{SOURCE82} \
+   %{SOURCE50} \
+   %{SOURCE60} \
+   %{SOURCE80} %{SOURCE81} %{SOURCE82} %{SOURCE82} \
```

Comment:
shouldn't this have SOURCE83 at the end, not two SOURCE82s?

Reply:
That's why we need PRs, thank you for noticing!

Reply:
New buddy build after changing this:
[618523](https://dev.azure.com/mariner-org/mariner/_build/results?buildId=618523&view=results)